20-minimal/Dockerfile.c8s

```diff
@@ -0,0 +1,68 @@
+FROM quay.io/sclorg/s2i-core-c8s
```

Comment:
It should be UBI-minimal, because of nodejs-minimal images are built based on UBI-minimal images.

Reply:
I cannot use ubi (RHEL) when I want to install CentOS Stream packages. We could create our own sclorg minimal variant for CentOS Stream (because CentOS Stream itself does not produce one AFAICT), but I don't think it's necessary. I thought that testing the minimal image based on s2i-core should be enough to get some feedback.